.github/scripts/website.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e
+
+# install and build project
+npm install && npm run bootstrap && npm run build
```

Comment:
Kinda unfortunate this all this is necessary now just to deploy the website. This is just because we need to run docgen from the contracts folder?